README.md

```diff
@@ -79,20 +95,28 @@ You may use this library in your applications with the following dependency info
 
 - **In SBT**:
 
-    **Scala 2.11**
-
-    ``` SBT
-    // https://mvnrepository.com/artifact/io.github.spark-redshift-community/spark-redshift
-    libraryDependencies += "io.github.spark-redshift-community" %% "spark-redshift" % "4.0.0"
+    ```SBT
+    libraryDependencies += "io.github.spark-redshift-community" %% "spark-redshift_2.11" % "4.0.0"
```

Comment:
I haven't actually with SBT (I'm using the "without build tool" approach), but I'm assuming artifact ID does not depend on the build tool used. Same goes for the next few changes, eg `4.0.0` vs `master` for version.